mailbox/api/src/main/java/org/apache/james/mailbox/FlagsBuilder.java

```diff
@@ -23,6 +23,10 @@
 
 public class FlagsBuilder {
 
+    public static FlagsBuilder builder() {
+        return new FlagsBuilder();
+    }
+
     private final Flags internalFlags;
 
     public FlagsBuilder() {
```

Comment:
If we don't have the time to do the change, the best choice is probably not doing it at all.
Do you really think anybody will handle the second half of the refactoring in some weeks ? IMO we will stay in a middle ground ...